app/views/devise/registrations/edit.html.erb

```diff
@@ -5,22 +5,22 @@
     <%= devise_error_messages! %>
 
     <h3>Change your account details</h3>
-    <%= f.email_field :email, autofocus: true %>
+    <%= f.email_field :email %>
```

Comment:
hey @CamLatimer -- can you restore these extra attributes? We want to take autocomplete off the pregnancy forms, but we do want to make sure it stays on the account edit form.